### DISCUSSION TITLE
Direct frequency input using an encoder for devices without a touch screen.

### DISCUSSION THREAD

**Discussion by Yurij20** [PE5PVB/TEF6686_ESP32#559] [Ideas]
I suggest making direct input of frequency, memory cell, etc.  in devices without a touch screen using an encoder.  To do this, I suggest going through the menu to the input keyboard, and rotating the encoder to scroll through the necessary keys on it and by clicking on it, press the necessary ones and thus enter the necessary data :)

**Comment by andimik** [comment]
This is too complicated.

A touchless device can **easily** tuned to a frequency with xdr-gtk or StationList. Just use a wifi connection. 

Or you store it on a memory preset.
